tools/seed-kopis/reschedule-dates.py

```
@@ -0,0 +1,198 @@
+    re.DOTALL,
+)
+
+DT = "%Y-%m-%d %H:%M:%S"
+D = "%Y-%m-%d"
+
+
+def parse_dt(s: str) -> datetime:
+    return datetime.strptime(s, DT)
+
+
+def main() -> None:
```

Ruff (0.15.15)

[warning] Too many statements (73 > 50)

(PLR0915)